Users.md

```diff
@@ -0,0 +1,44 @@
+# Users applying the MASVS and MSTG
+
+## Companies
+
+Currently the MASVS and MSTG are used by at least the following companies:
+
+- [Brewsec](https://brewsec.io/)
+- [Briskinfosec Technology and Consulting Pvt Ltd](https://www.briskinfosec.com/)
+- [Citadelo](https://citadelo.com/en/blog/how-to-order-a-pen-test/)
+- [Comsec](https://comsecglobal.com/)
+- [continuumsecurity](https://continuumsecurity.net)
+- [Cyber Ninjas](https://www.CyberNinjas.com)
+- [FH Münster - University of applied sciences](https://www.fh-muenster.de)
+- [Genexus](https://www.genexus.com) & [Genexus Consulting](https://www.genexusconsulting.com/es/)
+- [hackenproof](https://hackenproof.com)
+- [Infosec](https://Infosec.com.br)
+- [Netguru](https://www.netguru.co/)
+- [NowSecure](https://www.nowsecure.com/)
+- [NVISO CVBA](https://www.nviso.be)
+- [Randorisec](https://randorisec.fr/)
+- [Secarma](https://www.secarma.com/)
+- [VantagePoint](https://www.vantagepoint.sg)
+- [Xebia](https://xebia.com)
+- [Websec Canada](https://www.websec.ca/mobile-application-security)
+
+## Standards, frameworks and guidelines recommending the MASVS and MSTG
+
+Currently, the MASVS and MSTG are recommending the MASVS and MSTG:
```

Comment:
Currently, the MASVS and MSTG are recommending the MASVS and MSTG: -> Currently, the MASVS and MSTG are recommended by: